java-frontend/src/main/java/org/sonar/java/se/xproc/MethodBehavior.java

```diff
@@ -20,24 +20,18 @@
 package org.sonar.java.se.xproc;
 
 import com.google.common.collect.ImmutableList;
-import org.sonar.java.resolve.JavaSymbol;
-import org.sonar.java.se.ExplodedGraph;
-import org.sonar.java.se.checks.SECheck;
-import org.sonar.java.se.constraint.ConstraintsByDomain;
-import org.sonar.java.se.symbolicvalues.SymbolicValue;
-import org.sonar.plugins.java.api.semantic.Symbol;
-import org.sonar.plugins.java.api.semantic.Type;
-
-import javax.annotation.Nullable;
-
 import java.util.ArrayList;
 import java.util.LinkedHashSet;
 import java.util.List;
 import java.util.Set;
 import java.util.stream.Stream;
+import org.sonar.java.se.ExplodedGraph;
+import org.sonar.java.se.checks.SECheck;
+import org.sonar.java.se.constraint.ConstraintsByDomain;
+import org.sonar.java.se.symbolicvalues.SymbolicValue;
+import org.sonar.plugins.java.api.semantic.Type;
 
 public class MethodBehavior {
-  private Symbol.MethodSymbol methodSymbol;
   private boolean varArgs;
   private boolean isStaticMethod;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "isStaticMethod" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1068)